includes/class-widget.php

```diff
@@ -66,16 +66,29 @@ public function widget( $args, $instance ) {
 		}
 
 		echo '<div class="license">';
+
+		if ( empty( $instance['layout'] ) || 'modal' === $instance['layout'] ) {
 			echo sprintf(
-				'<p><button ' . ( $is_amp ? 'on="tap:republication-tracker-tool-modal"' : '' ) . ' name="%1$s" id="cc-btn" class="republication-tracker-tool-button">%1$s</button></p>',
+				'<p><button ' . ( $is_amp ? 'on="tap:republication-tracker-tool-modal"' : '' ) . ' name="%1$s" id="cc-btn" class="republication-tracker-tool-button modal">%1$s</button></p>',
 				esc_html__( 'Republish This Story', 'republication-tracker-tool' )
 			);
+		}
+
+		if ( 'page' === $instance['layout'] ) {
 			echo sprintf(
-				'<p><a class="license" rel="noreferrer license" target="_blank" href="%s"><img alt="%s" style="border-width:0" src="%s" /></a></p>',
-				REPUBLICATION_TRACKER_TOOL_LICENSES[ $license_key ]['url'],
-				esc_html__( 'Creative Commons License', 'republication-tracker-tool' ),
-				esc_url( plugin_dir_url( dirname( __FILE__ ) ) ) . 'assets/img/' . $license_key . '.png'
+				'<p><button name="%1$s" id="cc-btn" class="republication-tracker-tool-button page"><a href="%2$s">%1$s</a></button></p>',
```

Comment:
```suggestion
				'<p><button name="%1$s" id="cc-btn" class="republication-tracker-tool-button page" onclick="window.location.href=\'%2$s\'">%1$s</button></p>',
```

The `<a>` inside a `<button>` element is not valid HTML. It also produced styling with off colors:

![CleanShot 2025-02-20 at 11 44 11@2x](https://github.com/user-attachments/assets/43b24de4-3fe5-4ca2-80d0-eb71fff9d06d)

Reply:
Hum, I think it's acceptable to go with the onclick approach adding the `role="link"` to the `<button>`

Reply:
Cool! That is the `role` is a good addition :)